submissions/JuliaNovak/js-dom/script.js

```diff
@@ -0,0 +1,68 @@
+const methods = [
+    {
+        id: 1,
+        type: 'Array.prototype.filter()',
+        description: 'The filter() method creates a shallow copy of a portion of a given array, filtered down to just the elements from the given array that pass the test implemented by the provided function.',
+        img: './img/filter.png'
+    },
+    {
+        id: 2,
+        type: 'Array.prototype.find()',
+        description: 'The find() method returns the first element in the provided array that satisfies the provided testing function. If no values satisfy the testing function, undefined is returned.',
+        img: './img/find.png'
+    },
+    {
+        id: 3,
+        type: 'Array.prototype.forEach()',
+        description: 'The forEach() method executes a provided function once for each array element.',
+        img: './img/forEach.png'
+    },
+    {
+        id: 4,
+        type: 'Array.prototype.map()',
+        description: 'The map() method creates a new array populated with the results of calling a provided function on every element in the calling array.',
+        img: './img/map.png'
+    },
+    {
+        id: 5,
+        type: 'Array.prototype.push()',
+        description: 'The push() method adds one or more elements to the end of an array and returns the new length of the array.',
+        img: './img/push.png'
+    },
+    {
+        id: 6,
+        type: 'Array.prototype.toString()',
+        description: 'The toString() method returns a string representing the specified array and its elements.',
+        img: './img/toString.png'
+    },
+]
+
+const methodsList = document.getElementsByClassName('methods-list')[0]
+methods.forEach(el => {
+    const li = document.createElement('li')
```

Comment:
Read about [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset)
Add the id value to the dataset property of the element.

Reply:
could you please explain the benefit of implementing the dataset property over other properties like classList?

Reply:
Each of our links is unique, as is our dataset. So we can add a unique identifier to the dataset of the `li` element. Using the `addEventListener` on click that we assign to the menu, we define the selected item. We get the ID from its dataset. Then a corresponding element of the array is searched. We show the result to the user.